components/apply/PositionPreference.tsx

```diff
@@ -0,0 +1,65 @@
+import { FC, useState, useEffect } from "react";
+import SelectInput from "@components/common/SelectInput";
+import MEMBER_ROLES from "@constants/member-roles.json";
+
+type Props = {
+  onFirstChoiceChange: (value: string) => void;
+  onSecondChoiceChange: (value: string) => void;
+};
```

Comment:
Would it be better to pass both the state and the onChange as props so we don't have to declare more hooks in here? 

The prop `onFirstChoiceChange` and local `setFirstChoice` seems a bit overlapping to me. Could we make it work with just one state hook in `AppForm`?